python/dgllife/model/model_zoo/gin_predictor.py

```diff
@@ -123,5 +127,7 @@ def forward(self, g, categorical_node_feats, categorical_edge_feats):
             * B for the number of graphs in the batch
         """
         node_feats = self.gnn(g, categorical_node_feats, categorical_edge_feats)
+        if self.readout == 'skip' and self.predict is None:
```

Comment:
In such cases, why don't you directly use `GIN` rather than `GINPredictor`?

Reply:
done